lldb/source/Plugins/Language/Swift/SwiftOptionSet.cpp

```diff
@@ -167,6 +167,12 @@ bool lldb_private::formatters::swift::SwiftOptionSetSummaryProvider::
 
     for (auto val_name : *m_cases) {
       llvm::APInt case_value = val_name.first;
+      // Print single valued sets without using enclosing brackets.
+      if (case_value == value) {
+        ss << '.' << val_name.second;
```

Comment:
might be nice to also make this change on line 184 and 187 in a follow-up commit.